contracts/utils/math/Math.sol

```diff
@@ -406,6 +406,22 @@ library Math {
         }
     }
 
+    /**
+     * @dev Returns the modular exponentiation of the specified base,
+     * exponent and modulus (b^e % m)
+     *
+     * Requirements:
+     *
+     * - modulus can't be zero
+     * - result should be obtained successfully
+     */
+    function modExp(uint256 b, uint256 e, uint256 m) internal view returns (uint256) {
+        require(m != 0, "ModularExponentiation: Can't calculate for modulus equal to zero");
```

Comment:
We need to replace these with a custom error following [our guidelines](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/630844ef50c7581f8a91d15f535e9c67823951a5/GUIDELINES.md).

Reply:
Added custom error